.gitlab-ci.yml

```diff
@@ -14,16 +14,46 @@ cache:
     - assembl/static2/node_modules/
     - /usr/local/share/.cache/yarn/v2
 
-test_frontend:
-  except:
-    - master
-  image: node:8
+# test_frontend:
+#   except:
+#     - master
+#   image: node:8
+#   before_script:
+#     - export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
+#     - cd assembl/static2
+#     - yarn
+#   script:
+#     - yarn run test
+#     - yarn run eslint
+#     - yarn run stylelint
+#     - yarn run flow --quiet
+
+test_integration_and_ui:
+  image: node:8-slim
   before_script:
-    - export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-    - cd assembl/static2
+    - apt-get update -qq
+    # install chrome dependencies
+    - apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
+    # install git
+    - apt-get install -yq git
```

Comment:
Use a bash script for the git setup. We need to reuse it for the push_to_master job...

Reply:
Je peux le faire si tu veux mais en fin de compte je n'ai plus besoin d'installer git pour ce job (car on peut utiliser l'image docker node:8 en fin de compte) donc je pense que ça aurait sa place sur une autre branche.

Reply:
ok